webv2/profile/profile.go

```diff
@@ -201,6 +203,57 @@ func setConnectionProfile(isSource bool, sessionState session.SessionState, req
 
 }
 
+func CleanUpStreamingJobs(w http.ResponseWriter, r *http.Request) {
```

Comment:
Clean up should be applicable to both command line mode and web UI mode and looks like a backend functionality. Is `webv2` the correct location to write this logic?

Reply:
Can we also validate if cleanup will trigger as part of low downtime migrations triggered via the command line mode as well?

Reply:
We are not cleaning this up as part of command line because we are relying on user input when they think that migration is complete and they are ready to stop the writes to source database for some time

Reply:
Got it. Can we move this mode outside of `webv2` though? Looks more suited to `streaming.go` IMO.

Reply:
done